Gemfile

```diff
@@ -12,8 +12,10 @@ end
 gem 'rails', '~> 5.0.1'
 gem 'rails-controller-testing'
 
-gem 'sprockets', '~> 3.6.3'
-gem 'sprockets-rails', '~> 3.1.1'
+# Compile Sprockets assets concurrently in `assets:precompile`.
+# Ref: https://github.com/rails/sprockets/pull/469
+gem 'sprockets', github: 'wjordan/sprockets', ref: 'concurrent_asset_bundle_3.x'
```

Comment:
I think code-dot-org is probably better to use in general, but if there's an expectation that it is short-lived, I don't think I care a lot.